src/components/ListItems.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-nested-ternary */
```

Comment:
Let's remove all disabled Linters, Linting helps to achieve cleaner and more professional code ✊

Kindly check for other occurrences 🙂

Reply:
Done